src/React/Resources/assets/src/render_controller.ts

```diff
@@ -27,6 +27,10 @@ export default class extends Controller {
 
         this._dispatchEvent('react:connect', { component: this.componentValue, props: props });
 
+        if (!this.componentValue) {
+            throw new Error('No component specified for react controller');
+        }
+
```

Comment:
This is just to solve the unitilized error that Typescript raise when compiling assets, as it seems to be related to stimulus code -- we don't initialise it here but I guess it would in the runtime